runner/utils/driver/docker/docker.go

```diff
@@ -338,7 +341,7 @@ func (d *Docker) DeleteService(ctx context.Context, projectID, serviceID, versio
 	hostFile.RemoveHost(utils.GetInternalServiceDomain(projectID, serviceID, version))
 
 	// Get if current version was the last remaining service
-	isLastContainer, err := d.checkIfLastService(ctx, projectID, serviceID)
+	isLastContainer, err := d.checkIfLastService(ctx, projectID, serviceID, d.clusterID)
```

Comment:
No need to pass d.Clusterid you have access to it inside the function as it is a method